src/builtin/capitalize.h

```diff
@@ -0,0 +1,90 @@
+/**
+ *  capitalize.h
+ *
+ *  Built-in "|capitaluze:bool" modifier
+ *
+ *  @author Youri Moll <youri.moll@copernica.com>
+ *  @copyright 2019 Copernica BV
+ */
+
+/**
+ *  Namespace
+ */
+namespace SmartTpl { namespace Internal {
+
+/**
+ *  Class definition
+ */
+class CapitalizeModifier : public Modifier
+{
+public:
+    /**
+     *  Destructor
+     */
+    virtual ~CapitalizeModifier() {};
+
+    /**
+     *  Modify a value object
+     *  @param  input
+     *  @param  params      Parameters used for this modification
+     *  @return Value
+     */
+    VariantValue modify(const Value &input, const SmartTpl::Parameters &params) override
+    {
+        // copy the entire string
+        std::string output(input.toString());   
+    
+        // our delimiters, we will capitalize after any of these characters are encountered
+        const std::string delims = " \t\r\n\f\v";
```

Comment:
Can you use `const char *` here, and use `strchr(c, delims)` to check if a character is a delimiter?